src/neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -421,6 +421,7 @@ public StackItem ToStackItem(ReferenceCounter referenceCounter)
                 NetworkFee,
                 ValidUntilBlock,
                 Script,
+                new Array(referenceCounter, Signers.Select(u=> (ByteString)u.ToArray()))
```

Comment:
How to deserialize it in a contract?

Reply:
Only the address better?

Reply:
@csmuller Does it work for you?

Reply:
I think we should answer the question first: why do we need to read the signers in a contract?

Reply:
To implement some checks in `verify`?

Reply:
Maybe we can create some syscalls to handle it? Just like `CheckWitness`?